packages/database/src/migrations/20210602041852-Rename-lesmis-permission-groups-modifies-data.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const updateLESMISReportPermissionGroup = function (db, oldName, newName) {
+  return db.runSql(`
+  UPDATE "permission_group"
+  SET "name" = '${newName}'
+  WHERE "name" = '${oldName}'
+  `);
+};
+
+exports.up = async function (db) {
+  await updateLESMISReportPermissionGroup(db, 'Laos Schools Admin', 'LESMIS Admin');
+  await updateLESMISReportPermissionGroup(db, 'Laos Schools Super User', 'LESMIS User');
```

Comment:
There's actually a bit more to do here, as well as renaming we need to restructure the hierarchy so that each one has its parent set to the one above

Reply:
Ooops, done in a later commit!